voter_info/voter_info/settings.py

```diff
@@ -25,12 +27,15 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 if IN_PRODUCTION:
     SECRET_KEY = os.getenv('DJANGO_SECRET_KEY', None)
-    DEBUG = False
+    DEBUG = os.getenv('DEBUG', False)
+    RAVEN_CONFIG = {
+        'dsn': os.getenv('SENTRY_DSN', None),
+    }
 else:
     DEBUG = True
     SECRET_KEY = '270i7+@=r$sc#1hv!6#lkl6j+fhd8cwvn6$^ijk4q6l#0d&nu1'
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['*']
```

Comment:
This is probably ok, but can leave one open to some security issues. I think the nature of our product means we shouldn't have to worry about it, though.

https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-ALLOWED_HOSTS
https://docs.djangoproject.com/en/2.1/topics/security/#host-headers-virtual-hosting